modules/ROOT/pages/fd-project-history.adoc

```diff
@@ -0,0 +1,41 @@
+= Discover What’s Changed in a Project
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords:
+
+:imagesdir: ../assets/images
+
+You can see a history of changes you make to a project in Flow Designer.
+These changes are automatically saved every few minutes while you are working on your project.
```

Comment:
I recommend deleting this line.  Actually, the changes are auto-saved every few seconds, not every few minutes.  But in the release after this one, we will remove auto-save so that all changes happen when there are explicit "Save" and "Cancel" buttons that cause the changes.  That's why I think it's better not to say anything here.

Reply:
Okay, will remove.

Reply:
Fixed.